addons/web_editor/static/src/js/wysiwyg/widgets/media.js

```diff
@@ -934,7 +931,7 @@ var IconWidget = SearchableMediaWidget.extend({
      * @override
      */
     _clear: function () {
-        var allFaClasses = /(^|\s)(fa(\s|$)|fa-[^\s]*)/g;
+        var allFaClasses = /(^|\s)(fa|fa-\S*)(?=\s|$)/g;
```

Comment:
Note: I did not commented the other ones for the sake of not changing useless stuff in stable version but since you are adding change here: cannot `(?=\s|$)` and `(^|\s)` be replaced by a simple `\b` ?

Reply:
This is not a useless change, most of these regex were somewhat broken and would not catch some cases when the regex matches multiple classes and the classes are adjacent to each other. In particular, because this regex requires a space or start of string, but also matches the space after `fa`, it would miss an fa-class if it's right behind the `fa`. `\b` doesn't work because it matches on dashes (ie: `[\b]fa[\b]-[\b]search[\b]`)

Reply:
I did not say it is an useless change, I said the exact opposite: that I did not commented the other regexes for the potential `\b` change to avoid you making useless change (the \b change) for no reason.

All good then.

Reply:
Ah gotcha!